EDITORIAL_GUIDELINES.md

```diff
@@ -0,0 +1,167 @@
+            &nbsp;&nbsp; **Date/Time Format**<br>
+        </td>
+        <td>
+            - Bold <br>
+            - Use the display name in the non-normative section.<br>
+            - The first occurrence in a section is linked to the section.
+        </td>
+    </tr>
+    <tr>
+        <td><strong>Columns &amp; Attributes IDs:</strong></td>
+        <td>
+           <strong>Columns IDs</strong>:<br>
+            - PricingQuantity</strong><br> 
+            - PricingUnit</strong><br> 
+            - ProviderName</strong> <br><br>
+          <strong>Attributes IDs</strong>:<br>
+            - CurrencyCodeFormat <br> 
+            - DateTimeFormat <br>
+        </td>
+        <td>
+          <strong>Columns IDs:</strong>:<br>
+          &nbsp;&nbsp; PricingQuantity <br>
+          &nbsp;&nbsp; PricingUnit</strong><br>
+          &nbsp;&nbsp; ProviderName</strong> <br><br>
+          <strong>Attributes IDs:</strong> </br>
+          &nbsp;&nbsp; CurrencyCodeFormat </br>
+          &nbsp;&nbsp; DateTimeFormat <br>
+        </td>
+        <td>
+           - Use PascalCamel case (the first letter of every word, is capitalized)
+           - Normal text without bold or italics.<br>
+           - The first occurrence in a section is linked to the section.
+        </td>
+    </tr>
+    <tr>
+        <td><strong>Normative Keywords &amp; Statements</strong></td>
+        <td>
+            MUST, MAY, MUST NOT and normative statements
+        </td>
+        <td>
+            This column:</br>
+            &nbsp;&nbsp; * MUST NOT be null when ChargeClass is ... </br>
+            &nbsp;&nbsp; * MUST be null when ChargeCategory is ... </br>
+            &nbsp;&nbsp; * MAY be null for all other combinations of ... </br>
+        </td>
+        <td>
+           - All uppercase, without bold.<br>
+           - Bullet list format. <br>
+        </td>
+    </tr>
+    <tr>
+        <td><strong>Glossary</strong></td>
+        <td>
+            [SKU](), [resource](), [service]()
+        </td>
+        <td>
+            [*SKU*](#glossary:sku) <br>
+            [*resource*](#glossary:resource) <br>
+            [*service*](#glossary:service) <br>
+        </td>
+        <td>
+            - Blue font + italic<br>
+            - The first ocurrence in a section is linked to the glossary.
+        </td>
+    </tr>
+    <tr>
+        <td><strong>Important Text</strong></td>
+        <td> <img width="492" alt="image" src="https://github.com/user-attachments/assets/c6f60ff9-1503-43a3-8229-004595b334d2"></td>
+        <td>> Important Consideration</td>
+        <td>- It is added as a note.</td>
+    </tr>
+    <tr>
+        <td><strong>Key-Value Format</strong></td>
+        <td>JSON Script</td>
+        <td>
+            <pre>
+              **Example**:
+              ```json
+                {
+                    "key1": "value1",
+                    "key2": true,
+                    "key3": 123
+                }
+              ```
+            </pre>
```

Comment:
This pre tag is causing excessive indentation in the table. Adjusting for that.

We should probably call out how we want indentation to work for code samples. Specifically, 2 vs. 4 spaces and there should not be any initial indentation. For instance, the original example had 2 extra spaces.
````suggestion
            <pre>
**Example**:
```json
{
    "key1": "value1",
    "key2": true,
    "key3": 123
}
```
            </pre>
````